Exercice_2_-_Le_chiffre_de_Cesar/caesar.py

```diff
@@ -80,56 +80,30 @@
 >>> chr(90+3)
 ']'
 """
+import string
+import logging
+import sys
 
+class Colors:
+    """Colored prints
```

Comment:
Alors la règle pour les docstring c'est :

- si ça tient sur une ligne on met les `"""` au début et à la fin de la-dite ligne
- sinon chaque `"""` est sur sa propre ligne
- Une docstring commence par un verbe à la forme impérative et se termine par un point : _Print your mama._, _Tell the hour_